### PR TITLE
ARROW-4401: [Python] Alpine dockerfile fails to build because pandas requires numpy as build dependency

### PR DESCRIPTION
**Pull request by kszucs**
It's enough to install them sequentially: [kszucs/crossbow/build-433](https://github.com/kszucs/crossbow/branches/all?utf8=%E2%9C%93&query=build-433)

### CODE REVIEW

**Comment by kszucs** [comment]
Now it raises an easy_install error..., however it works on ubuntu. I don't know why easy_install gets triggered at all.

File: docker-compose.yml [hunk 1]
```diff
@@ -19,14 +19,22 @@
 
 version: '3.5'
 
+# TODO(kszucs): set arrow's mount to :ro mode, once all of the builds are
```
Comment (kszucs) [inline docker-compose.yml]: @wesm I had to disable the read-only mount, because it has caused tiny, but really annoying issues.

Comment (wesm) [reply docker-compose.yml]: No problem :)

**Review by wesm — APPROVED** [review]
+1, thanks @kszucs!

**Pull request closed without merging**